src/cargo/util/toml/mod.rs

```diff
@@ -2151,6 +2164,40 @@ fn to_dependency_source_id<P: ResolveToPath + Clone>(
     }
 }
 
+pub(crate) fn lookup_path_base(
+    base: &str,
+    gctx: &GlobalContext,
+    manifest_root: &Path,
+) -> CargoResult<PathBuf> {
+    if !gctx.cli_unstable().path_bases {
+        bail!("usage of path bases requires `-Z path-bases`");
+    }
```

Comment:
Can we use this instead
```rust
    gctx.cli_unstable().fail_if_stable_command(
        gctx,
        "clean gc",
        12633,
        "gc",
        gctx.cli_unstable().gc,
    )?;
```

Reply:
Done

Reply:
I added a new `fail_if_stable_feature` function that prints ``the `{feature}` feature is unstable`` otherwise it was a bit weird to see ``the `cargo path bases` command`` in the error message.

Reply:
Switched to using a feature instead of a `-Z`